client/src/components/Workbench/Workbench.js

```diff
@@ -20,6 +20,7 @@ import './Workbench.scss';
 function Workbench() {
   const { path } = useRouteMatch();
   const dispatch = useDispatch();
+  const hideApps = useSelector(state => state.workbench.config.hideApps);
```

Comment:
Still works, but I had meant it could be combined in this `useSelector` block [here](https://github.com/TACC/Core-Portal/blob/f6fba7aee7b5553a96c3b685f2dc752aed9d7ca9/client/src/components/Workbench/Workbench.js#L26)

Reply:
Thanks for pointing that out.